bigtable/spark/project/build.properties

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+sbt.version = 1.3.13
```

Comment:
Is this tracking the same version as spark? (latest is 1.4.4 according to https://www.scala-sbt.org/download.html)

Reply:
I need to double check - this may have just been the latest at the time. For now, let's leave it as is.